package.json

```diff
@@ -84,23 +84,23 @@
     "@types/random-js": "1.0.30",
     "all-contributors-cli": "5.4.1",
     "del-cli": "1.1.0",
-    "fs-extra": "7.0.0",
-    "handlebars": "4.0.11",
+    "fs-extra": "^7.0.0",
+    "handlebars": "^4.1.2",
     "husky": "1.1.2",
     "jest": "24.7.1",
     "jest-fetch-mock": "1.6.5",
     "ncp": "2.0.0",
     "prettier": "1.10.2",
-    "release-it": "7.4.8",
+    "release-it": "^10.4.4",
```

Comment:
Let's not update the version, also I prefer installing an exact version without `^`

Reply:
Okay we will revert it back to the old versions.